registration/templates/templatetags/basic_attendee_form.html

```diff
@@ -101,11 +113,11 @@
     <label for="birthDate" class="col-sm-3 control-label">Date of Birth <span style="color:red;">*</span></label>
     <div class="col-sm-9">
         <div class="col-sm-3">
-          <input type="number" name="byear" id="byear" class="form-control" placeholder="Year" value="{{attendee.birthdate|date:"Y"}}" max="{% now "Y" %}" min="1900" maxlength="4" required>
+          <input type="number" name="byear" id="byear" class="form-control" placeholder="Year" value="{{attendee.birthdate|date:"Y"}}" max="{% now "Y" %}" min="1900" maxlength="4" required autocomplete="bday-year" />
         </div>
 
         <div class="col-sm-5">
-          <select id="bmonth" name="bmonth" class="form-control" required>
+          <select autocomplete="bday-month" class="form-control" id="bmonth" name="bmonth" required>
```

Comment:
Personally, I don't like to alphabetize properties like this.  The previous ordering places the properties most pertinent to DOM manipulation first, making it a little easier to reference.